Scrabble.java

```diff
@@ -85,10 +114,23 @@ public static void playHand(String hand) {
 			// non-whitespace characters. Whitespace is either space characters, or  
 			// end-of-line characters.
 			String input = in.readString();
-			//// Replace the following break statement with code
-			//// that completes the hand playing loop
-			break;
+
+			if (input.equals(".")) {
+				break;
+			}
+
+			input = input.toLowerCase(); // normalizing input to lowercase letters
+
+			if (isWordInDictionary(input) && MyString.subsetOf(input, hand)) {
+				score += wordScore(input);
+				// System.out.println("'" + input + "' -> score: " + score);
+				System.out.println(input + " earned " + wordScore(input) + " points. Score: " + score + " points\n");

```

Comment:
why `\n` if you have used `println`

Reply:
There was an issue with autograding which this addition fixed

Reply:
weird .. please let know if happens in future. 
This is not ideal